src/lib/y2security/selinux.rb

```diff
@@ -122,21 +188,31 @@ def save
       return false unless configurable?
 
       Yast::Bootloader.modify_kernel_params(mode.options)
+      config_file.selinux = mode.id.to_s
+      config_file.save
 
       return true if Yast::Mode.installation
 
       Yast::Bootloader.Write
     end
 
+    # Returns needed patterns defined in the product features
+    #
+    # @return [Array<Sring>] collection of defined patterns in product features to have
+    #                        SELinux working as expected
+    def needed_patterns
+      product_feature_settings[:patterns].to_s.split
```

Comment:
ah, what about that disabled mode? I am not adapting firewall code and it basically always select it even when selinux is disabled.

Reply:
Addressed in https://github.com/yast/yast-security/pull/89, thanks for catching.